src/main/java/com/odysseusinc/athena/api/v1/controller/VocabularyController.java

```diff
@@ -91,38 +93,32 @@
 public class VocabularyController {
     private static final Logger LOGGER = LoggerFactory.getLogger(VocabularyController.class);
 
-    private VocabularyService vocabularyService;
-    private VocabularyConversionService vocabularyConversionService;
-    private LicenseRequestSender licenseRequestSender;
-    private UserService userService;
-    private DownloadBundleService downloadBundleService;
-    private FileHelper fileHelper;
-    private ConverterUtils converterUtils;
+    private final ConverterUtils converterUtils;
+    private final DownloadBundleService downloadBundleService;
+    private final FileHelper fileHelper;
+    private final LicenseRequestSender licenseRequestSender;
+    private final UserService userService;
+    private final VocabularyConversionService vocabularyConversionService;
+    private final VocabularyService vocabularyService;
 
     @Autowired
     @SuppressWarnings("SpringJavaAutowiringInspection")
-    public VocabularyController(VocabularyService vocabularyService,
-                                VocabularyConversionService vocabularyConversionService,
-                                UserService userService,
-                                DownloadBundleService downloadBundleService,
-                                FileHelper fileHelper,
-                                ConverterUtils converterUtils,
-                                LicenseRequestSender licenseRequestSender) {
-
-        this.vocabularyService = vocabularyService;
-        this.vocabularyConversionService = vocabularyConversionService;
-        this.userService = userService;
+    public VocabularyController(ConverterUtils converterUtils, DownloadBundleService downloadBundleService, FileHelper fileHelper, LicenseRequestSender licenseRequestSender, UserService userService, VocabularyConversionService vocabularyConversionService, VocabularyService vocabularyService) {
+        this.converterUtils = converterUtils;
         this.downloadBundleService = downloadBundleService;
         this.fileHelper = fileHelper;
-        this.converterUtils = converterUtils;
         this.licenseRequestSender = licenseRequestSender;
+        this.userService = userService;
+        this.vocabularyConversionService = vocabularyConversionService;
+        this.vocabularyService = vocabularyService;
     }
 
+
     @ApiOperation("Get vocabularies.")
     @RequestMapping(produces = MediaType.APPLICATION_JSON_VALUE, method = RequestMethod.GET)
-    public ResponseEntity<List<UserVocabularyDTO>> getAllForCurrentUser() throws Exception {
+    public ResponseEntity<List<UserVocabularyDTO>> getAllForCurrentUser() {
```

Comment:
Well, while code has refactored why not to simply return entity w/o redundant wrappers such as `ResponseEntity`?